px-spinner.html

```diff
@@ -88,7 +88,18 @@
             value: function(){
               return Px.Modernizr.smil;
             }
-          }
+          },
+          /**
+           * This property is to control whether the spinner is rendered as svg or div element.
+           * If useDiv is false, which is by default value, spinner is rendered as svg,
+           * otherwise the spinner is rendered as div.
+           *
+           */
+          useDiv: {
```

Comment:
I don't think is the right name. It doesn't really indicate what it does, and being a div isn't what makes the animation work when the event loop is blocked.

Maybe `disable-svg-animation` would be clearer.

Reply:
Sounds good, updated, thanks.